src/select/index.ts

```diff
@@ -149,6 +157,40 @@ export class Select extends ThemedMixin(FocusMixin(WidgetBase))<SelectProperties
 		option && onChange && onChange(option, key);
 	}
 
+	private _validate() {
+		const {
+			_state: state,
+			properties: { onValidate, value, required }
+		} = this;
+
+		if (!onValidate) {
+			return;
+		}
+
+		if (
```

Comment:
if this widget has no value, but it is required then should it not report as `invalid` once it's been interacted with?

Reply:
This has been addressed. I have also matched the logic to be closer to TextInput and TextArea.